app/src/main/res/menu/drawer_view.xml

```diff
@@ -35,11 +35,11 @@
             android:id="@+id/nav_announcement"
             android:icon="@drawable/ic_announcement_black_24dp"
             android:title="@string/title_nav_announcement"/>
-    </group>
```

Comment:
why have you done this?

Reply:
for systematic order because after this group all are in dialogue boxes. So i shifted usage activity to the above group

Reply:
revert this because this will allow usages nav item to be checkable and this is not required.